src/components/Button/Button.tsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import {
+  ActivityIndicator,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+} from 'react-native';
+import {ButtonProps} from './ButtonProps.types';
+import {scale, verticalScale} from '../../utils';
+import {useThemeColor} from '../../hooks';
+import Icon from 'react-native-vector-icons/Ionicons';
```

Comment:
Why are we using this Icon and not the CustomIcon component created?

Reply:
Custom Icon is for the iconset we have added using icomoon and for the Ionicons i am using it in this way.